python/pyxet/pyxet/cli.py

```diff
@@ -135,6 +135,7 @@ def sync(source: Annotated[str, typer.Argument(help="Source folder to sync")],
              target: Annotated[str, typer.Argument(help="Target location of the folder")],
              use_mtime: Annotated[bool, typer.Option("--use-mtime", help="Use mtime as criteria for sync")] = False,
              message: Annotated[str, typer.Option("--message", "-m", help="A commit message")] = "",
+             update_size: Annotated[bool, typer.Option("--update-size", hidden=True)] = False,
```

Comment:
[nit] for posterity, it would be nice to document why we added this flag and what it's used for